### PR TITLE
Replaced DraggableCard based on Figma

### PR DESCRIPTION
**Pull request by maheshwaran01m**
- Added dark mode for checkBox checked
 - Replaced DraggableCard based on Figma
 
Video:

https://github.com/tarkalabs/tarka-ui-kit-ios/assets/102943217/d690c7d6-4a67-4875-9888-fe71687fb9d7

https://github.com/tarkalabs/tarka-ui-kit-ios/assets/102943217/1282e4b9-1b90-4a19-877f-8c21c68d2035

### CODE REVIEW

**Review by arvindhsukumar — APPROVED** [review]

File: Sources/TarkaUI/Components/DraggableCard/TUIDraggableCard.swift [hunk 1]
```diff
@@ -1,69 +1,104 @@
...
-  ///
-  /// - Parameters:
-  ///   - content: A closure that returns the content to display in the card.
-  ///
-  public init(@ViewBuilder content: @escaping () -> Content) {
-    self.content = content
+  private let title: String
+  private var action: () -> Void
+  
+  private var toggleSelected: Bool
```
Comment (gopinathcholan) [inline Sources/TarkaUI/Components/DraggableCard/TUIDraggableCard.swift:31]: pls add `is` as prefix for consistency
```suggestion
  private var isToggleSelected: Bool
```

File: Sources/TarkaUI/Components/DraggableCard/TUIDraggableCard.swift [hunk 2]
```diff
@@ -1,69 +1,104 @@
...
+      .contentShape(.dragPreview, contentShape)
+      .onDrag({
+        action()
+        return .init()
+      }, preview: previewItemView)
+  }
+  
+  private func previewItemView() -> some View {
+    Color.clear
+      .frame(width: 1, height: 1)
```
Comment (gopinathcholan) [inline Sources/TarkaUI/Components/DraggableCard/TUIDraggableCard.swift:66]: Could you add comments why it is for

File: Sources/TarkaUI/Extensions/Image+Extensions.swift [hunk 3]
```diff
@@ -17,3 +17,18 @@ public extension Image {
...
+  }
+}
+
+public enum Icon: String {
+  case checkBoxChecked = "checkbox_checked"
+  case checkBoxUnChecked = "checkbox_unchecked"
+  case toggleSwitchOn = "toggle_switch_on"
+  case toggleSwitchOff = "toggle_switch_off"
+  case toggleSwitchDisabledOff = "toggle_switch_diabled_off"
+  case toggleSwitchDisabledOn = "toggle_switch_diabled_on"
```
Comment (gopinathcholan) [inline Sources/TarkaUI/Extensions/Image+Extensions.swift:33]: why image assets are introduced..Are those not available in FluentIcons?

File: Sources/TarkaUI/Components/DraggableCard/TUIDraggableCard.swift (hunk 2 above)
Comment (maheshwaran01m) [reply Sources/TarkaUI/Components/DraggableCard/TUIDraggableCard.swift:67]: @gopinathcholan  Updated

File: Sources/TarkaUI/Extensions/Image+Extensions.swift (hunk 3 above)
Comment (maheshwaran01m) [reply Sources/TarkaUI/Extensions/Image+Extensions.swift:33]: yeah, it's custom one

Comment (gopinathcholan) [reply Sources/TarkaUI/Extensions/Image+Extensions.swift:33]: Did you check this with Yogesh?

Comment (maheshwaran01m) [reply Sources/TarkaUI/Extensions/Image+Extensions.swift]: @gopinathcholan Updated toggleSwitch

**Review by gopinathcholan — APPROVED** [review]

**Pull request merged**